Cargo.toml

```diff
@@ -6,18 +6,16 @@ license = "MIT"
 documentation = "https://aurora0001.github.io/rawr/doc/rawr/index.html"
 repository = "https://github.com/Aurora0001/rawr"
 description = "Rust API Wrapper for Reddit"
-build = "build.rs"
 exclude = [
   "**/*.out",
   "**/*.lock",
   "**/*.yml",
   "**/*.sh"
 ]
 
-[build-dependencies]
-serde_codegen = "0.8"
-
 [dependencies]
-hyper = "0.9.10"
-serde = "0.7.14"
-serde_json = "0.7.4"
+hyper = "0.10"
+hyper-native-tls = "0.2"
```

Comment:
Doesn't compile on machines with OpenSSL 1.1+ (same as rawr without this PR), simply changing it to "0.3" worked very well for me!